config/localTemplate.go

```diff
@@ -74,7 +74,7 @@ type Local struct {
 	CadaverSizeTarget uint64 `version[0]:"1073741824"`
 
 	// IncomingConnectionsLimit specifies the max number of long-lived incoming
-	// connections.  0 means no connections allowed.  -1 is unbounded.
+	// connections. 0 means no connections allowed. Must be non-negative.
```

Comment:
It seems strange to say `Must be non-negative` and also have the version 0 default be -1. I understand that you're supposed to define all values in config.json, but what happens if you were to omit this? Would it default to "800"?

Reply:
Omit `version[0]:"-1"`? Then upgrades from version 0 will not work. I think, this is nothing to worry about.

Reply:
I think that upgrades from version 0 would still work. It's only an issue if someone actively specifying -1, which is unlikely.